.travis.yml

```diff
@@ -1,3 +1,7 @@
+os:
+  - linux
+  - osx
+  - windows
```

Comment:
The expressjs project currently only tests against different operating systems if there is some kind of os-dependent code path like file operations or path manipulation. Does this apply to this project, and if so, please elaborate on how :+1:

Reply:
See the travis log its breaking against windows operating system. 
Here is the logfile. 

Downloading bootstrap node from https://nodejs.org/dist/v10.12.0/node-v10.12.0-win-x64.7z
######################################################################## 100.0%
The command "nvs add 1.8" failed and exited with 1 during .

Reply:
The expressjs project tests windows using AppVeyor and not using Travis CI.